client/src/pages/Apps/AppCards/AppCardItem/AppCardItem.js

```diff
@@ -32,12 +25,27 @@ const AppCardItem = ({ id, name, developer, type, description, images }) => {
             </header>
 
             <p className={styles.appCardDescription}>
-                {summarise(description)}
+                <Summarise>
+                    <ReactMarkdown
+                        allowedElements={['p']}
```

Comment:
This is used to remove tags like headers from the Summarised description on the front-page.